user_guide/intro_users_and_benefits.rst

```diff
@@ -0,0 +1,163 @@
+.. _oro-benefits:
+
+OroCRM Users and Their Benefits
+===============================
+
+This chapter of the guide explains the most common OroCRM user roles in order to help you understand which sections of 
+this user guide best match your work duties.
+
+Something for Everyone
+----------------------
+
+Collect and Process information easily
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+OroCRM handles critical CRM tasks such as:
+
+- Collect and aggregate information related to customers, sales, marketing campaigns, sales pipeline, and high level 
+  business performance. 
```

Comment:
Edit: - Collecting and aggregating information related to customers, sales, marketing campaigns, the sales pipeline, and high-level business performance.